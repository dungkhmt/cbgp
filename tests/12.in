40 40
40 57
13 29
29 37
37 8
8 27
27 39
39 10
10 13
23 3
3 5
5 17
17 11
11 24
24 16
16 22
22 34
34 35
35 23
1 15
15 6
6 38
38 28
28 32
32 2
2 19
19 1
9 33
33 31
31 14
14 26
26 40
40 25
25 9
21 7
7 4
4 36
36 21
12 20
20 18
18 30
30 12
30 20
28 6
14 2
29 3
32 11
38 5
18 4
16 27
34 39
33 1
35 19
37 17
24 8
26 9
23 10
21 12
31 7
1
9 9 35 9 19 17 7 0 21 19 23 21 44 0 27 11 8 7 18 14 31 10 0 0 20 14 37 7 22 22 21 10 25 15 5 0 11 9 3 2 5 5 16 10 17 15 28 10 9 8 12 7 22 11 24 20 22 14 3 0 38 6 34 10 6 6 14 10 12 10 8 1 24 14 22 20 17 13 11 8 
