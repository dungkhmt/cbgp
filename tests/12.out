(12, 28), (28, 36), (36, 7), (7, 26), (26, 38), (38, 9), (9, 12), (22, 2), (2, 4), (4, 16), (16, 10), (10, 23), (23, 15), (15, 21), (21, 33), (33, 34), (34, 22), (0, 14), (14, 5), (5, 37), (37, 27), (27, 31), (31, 1), (1, 18), (18, 0), (8, 32), (32, 30), (30, 13), (13, 25), (25, 39), (39, 24), (24, 8), (20, 6), (6, 3), (3, 35), (35, 20), (11, 19), (19, 17), (17, 29), (29, 11), (29, 19), (27, 5), (13, 1), (28, 2), (31, 10), (37, 4), (17, 3), (15, 26), (33, 38), (32, 0), (34, 18), (36, 16), (23, 7), (25, 8), (22, 9), (20, 11), (30, 6), 
0: (9, 9), 1: (35, 9), 2: (19, 17), 3: (7, 0), 4: (21, 19), 5: (23, 21), 6: (44, 0), 7: (27, 11), 8: (8, 7), 9: (18, 14), 10: (31, 10), 11: (0, 0), 12: (20, 14), 13: (37, 7), 14: (22, 22), 15: (21, 10), 16: (25, 15), 17: (5, 0), 18: (11, 9), 19: (3, 2), 20: (5, 5), 21: (16, 10), 22: (17, 15), 23: (28, 10), 24: (9, 8), 25: (12, 7), 26: (22, 11), 27: (24, 20), 28: (22, 14), 29: (3, 0), 30: (38, 6), 31: (34, 10), 32: (6, 6), 33: (14, 10), 34: (12, 10), 35: (8, 1), 36: (24, 14), 37: (22, 20), 38: (17, 13), 39: (11, 8), 
9 9 35 9 19 17 7 0 21 19 23 21 44 0 27 11 8 7 18 14 31 10 0 0 20 14 37 7 22 22 21 10 25 15 5 0 11 9 3 2 5 5 16 10 17 15 28 10 9 8 12 7 22 11 24 20 22 14 3 0 38 6 34 10 6 6 14 10 12 10 8 1 24 14 22 20 17 13 11 8 
intersectionCount=0
minAngle=0.12751
sumAngle=46.241
minEdgeLength=1.41421
sumEdgeLength=343.665
minDistance=0.707107
sumDistance=28439.8

0: (9, 9)
1: (35, 9)
2: (19, 17)
3: (7, 0)
4: (21, 19)
5: (23, 21)
6: (44, 0)
7: (27, 11)
8: (8, 7)
9: (18, 14)
10: (31, 10)
11: (0, 0)
12: (20, 14)
13: (37, 7)
14: (22, 22)
15: (21, 10)
16: (25, 15)
17: (5, 0)
18: (11, 9)
19: (3, 2)
20: (5, 5)
21: (16, 10)
22: (17, 15)
23: (28, 10)
24: (9, 8)
25: (12, 7)
26: (22, 11)
27: (24, 20)
28: (22, 14)
29: (3, 0)
30: (38, 6)
31: (34, 10)
32: (6, 6)
33: (14, 10)
34: (12, 10)
35: (8, 1)
36: (24, 14)
37: (22, 20)
38: (17, 13)
39: (11, 8)

Time taken: 0.0238428 s
